pkg/deploy/spindeploy/expose_service/transform.go

```diff
@@ -84,8 +84,14 @@ func (t *exposeTransformer) setStatusAndOverrideBaseUrl(ctx context.Context, ser
 
 // findStatusUrl returns the overrideBaseUrl or load balancer url, indicating if it came from overrideBaseUrl
 func (t *exposeTransformer) findStatusUrl(ctx context.Context, serviceName string, overrideUrlName string) (string, bool, error) {
+	var overrideUrlName_sanitized string
 	// ignore error, overrideBaseUrl may not be set in hal config
-	statusUrl, _ := t.svc.GetSpinnakerConfig().GetHalConfigPropString(ctx, overrideUrlName)
+	if overrideUrlName[len(overrideUrlName)-1:] == "/" {
```

Comment:
You should leverage `strings` and do `strings.HasPrefix(overrideUrlName, "/")`

```
if strings.HasPrefix(overrideUrlName, "/") {
    overrideUrlName = overrideUrlName[:len(overrideUrlName) - 1]
}
```

You could drop the else and the additional variable.